docker/bin/activate

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I can't add the `-e` because it's not a script that run once but it gets sourced. From the url in the comment


> The set -e option instructs bash to immediately exit if any command has a non-zero exit status. **You wouldn't want to set this for your command-line shell**, but in a script it's massively helpful.

If I add `-e` it adds it to the shell, not the script since the file is sourced, not executed.